packages/terra-time-input/lib/TimeInput.js

```diff
@@ -0,0 +1,160 @@
+
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
+
+var _react = require('react');
+
+var _react2 = _interopRequireDefault(_react);
+
+var _moment = require('moment');
+
+var _moment2 = _interopRequireDefault(_moment);
+
+var _reactTextMask = require('react-text-mask');
+
+var _reactTextMask2 = _interopRequireDefault(_reactTextMask);
+
+var _TimePipe = require('./TimePipe');
+
+var _TimePipe2 = _interopRequireDefault(_TimePipe);
+
+require('./TimeInput.scss');
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+function _objectWithoutProperties(obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; }
+
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+
+function _possibleConstructorReturn(self, call) { if (!self) { throw new ReferenceError("this hasn't been initialised - super() hasn't been called"); } return call && (typeof call === "object" || typeof call === "function") ? call : self; }
+
+function _inherits(subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function, not " + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : subClass.__proto__ = superClass; }
+
+var propTypes = {
+  /**
+   * A moment object to use as the default initial time.
+   */
+  defaultTime: _react.PropTypes.object,
+  /**
+  * A callback function to execute when a time value is entered.
+  */
+  onChange: _react.PropTypes.func
+};
+
+var defaultProps = {
+  defaultTime: null,
+  onChange: null
+};
+
+var TimeInput = function (_React$Component) {
+  _inherits(TimeInput, _React$Component);
+
+  _createClass(TimeInput, null, [{
+    key: 'userLocale',
+    value: function userLocale() {
+      // TODO: Get the locale from i18nProvider
+      return 'en-US';
+    }
+  }, {
+    key: 'timeFormat',
+    value: function timeFormat() {
+      // TODO: Get the time format translation.
+      return 'HH:mm';
```

Comment:
How well does this format work with times such as 3:29? When we uplifted moment in blue_steel, we had to change to h:mm format as hh:mm would look specifically for a two digit hour.